src/test/fetchCoins.test.js

```diff
@@ -0,0 +1,8 @@
+import fetchCoins from '../services/fetchCoins';
+
+describe('fetchCoins', () => {
+  it('should fetch coins data', async () => {
+    const coinsData = await fetchCoins();
+    expect(coinsData).toBeDefined();
+  });
+});
```

Comment:
- The same issue is applicable here. the tests is failing. please, fix them
- According to the project requirement, your app should have unit tests for several components of the app and some pure functions in Redux code.